From f41befa9121be253ad6e169f406255f540ac2991 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rn=20Kottmann?= <joern@apache.org>
Date: Thu, 3 Apr 2014 08:02:14 +0000
Subject: [PATCH] OPENNLP-670 seqCodec is not initialized in the default
 constructor which causes a NPE if trained with default seq codec. Thanks  to
 Vinh Khuc for providing a patch.

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1584277 13f79535-47bb-0310-9956-ffa450edef68
---
 .../java/opennlp/tools/namefind/TokenNameFinderFactory.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java b/opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java
index f5d365295..fbedca2f5 100644
--- a/opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java
+++ b/opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java
@@ -54,8 +54,8 @@ public class TokenNameFinderFactory extends BaseToolFactory {
    * of the resources.
    */
   public TokenNameFinderFactory() {
+    this.seqCodec = new BioCodec();
   }
-
   
   public TokenNameFinderFactory(byte[] featureGeneratorBytes, final Map<String, Object> resources,
       SequenceCodec<String> seqCodec) {
